pdr_backend/contract/test/test_predictoor_contract.py

```diff
@@ -56,9 +56,9 @@ def test_get_exchanges(feed_contract1):
 
 
 @enforce_types
-def test_get_stake_token(feed_contract1, web3_pp):
-    stake_token = feed_contract1.get_stake_token()
-    assert stake_token == web3_pp.OCEAN_address
+def test_get_USDC(feed_contract1, web3_pp):
+    USDC = feed_contract1.get_USDC()
```

Comment:
If the contract previously had the function `get_stake_token()` then why change it here? Let's minimize the changes.